frontend/src/components/layout/Layout.tsx

```diff
@@ -120,36 +122,38 @@ export const Layout = (props: Props) => {
         <InterviewRecruitment profile={profiles.data?.[0]} />
         {csatSurvey}
         {npsSurvey}
-        <header className={`${styles.header} ${darkClass}`}>
-          <div className={styles["logo-wrapper"]}>
-            <Link href={homePath}>
-              <a className={styles.logo}>
-                <img
-                  src={logo.src}
-                  alt=""
-                  className={styles.logomark}
-                  width={42}
-                />
-                <img
-                  src={logoType.src}
-                  alt={logoAlt}
-                  className={styles.logotype}
-                  height={20}
-                />
-              </a>
-            </Link>
+        <header>
+          <div className={`${styles.header} ${darkClass}`}>
+            <div className={styles["logo-wrapper"]}>
+              <Link href={homePath}>
+                <a className={styles.logo}>
+                  <img
+                    src={logo.src}
+                    alt=""
+                    className={styles.logomark}
+                    width={42}
+                  />
+                  <img
+                    src={logoType.src}
+                    alt={logoAlt}
+                    className={styles.logotype}
+                    height={20}
+                  />
+                </a>
+              </Link>
+            </div>
+            <div className={styles["nav-wrapper"]}>
+              <Navigation
+                theme={isDark ? "free" : "premium"}
+                toggle={handleToggle}
+              />
+            </div>
           </div>
-          <div className={styles["nav-wrapper"]}>
-            <Navigation />
-            {whatsNew}
-          </div>
-          <div className={styles["app-picker-wrapper"]}>
-            <AppPicker theme={isDark ? "free" : "premium"} />
-          </div>
-          <nav className={styles["user-menu-wrapper"]}>
-            <UserMenu />
-          </nav>
+
+          {/* separated from header to make use of z-index */}
+          <MobileNavigation active={mobileMenuState} />
```

Comment:
Pull contents of header and wrapped them with a header tag that doesn't have styling. It was causing issues with mobile nav - this makes it so I we can slide the mobile nav behind the main nav.